### PR TITLE
Added section to the Coding Guide's Constructor Function 

### PR DESCRIPTION
**Pull request by janeyx99**
The change had to do with using local variables rather than reading/writing multiple times.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@janeyx99, thanks for the pull request! Maintainers, we have a signed CLA from @janeyx99, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by ggetz** [comment]
Thanks @janeyx99 and congrats on your first Cesium contribution!

Please add yourself to `CONTRIBUTORS.md` and update `CHANGES.md` with a description of the changes you made.

Also if you include 

```
Fixes #XXX
```

with the issue number you are fixing, it will link the issue, and automatically close it when we merge this.

File: Documentation/Contributors/CodingGuide/README.md [hunk 1]
```diff
@@ -544,6 +544,21 @@ var p = new Cartesian3(1.0, 2.0, 3.0);
 p.x = 'Cesium'; // Changes x to a string, slows down property access
 ```
 
+* In a constructor function, consider properties as write once; do not write to them multiple times or read them. Create a local if they need to be ready. For example:
```
Comment (ggetz) [inline Documentation/Contributors/CodingGuide/README.md]: Just some nick pick on wording:

"do not write to them multiple times or read them" -> "do not write to them or read them multiple times"

"Create a local" -> "Create a local variable".

**Comment by ggetz** [comment]
Also - you'll want to leave your `master` branch the same as Cesium's `master` branch. When you make a change, create a new branch. I can show you how if you need help.

**Comment by ggetz** [comment]
Thanks @janeyx99! Merge master into this branch and this should be good to go!

**Comment by hpinkos** [comment]
@janeyx99 reminder to merge in master

**Comment by pjcozzi** [comment]
This PR is 25 days old.  Is it ready to merge?

**Comment by janeyx99** [comment]
Oh sorry! I didn't realize this wasn't done yet :( I think I've merged now?

**Comment by pjcozzi** [comment]
Thanks!

**Pull request merged**